### PR TITLE
Add a warning when ansible_key label is not found on a secret.

### PR DESCRIPTION
**Pull request by kaysond**
See #30

### CODE REVIEW

**Comment by kaysond** [comment]
Tested locally on 
```
ansible 2.10.3
  config file = None
  configured module search path = ['/home/administrator/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.8.5 (default, Jul 28 2020, 12:59:40) [GCC 9.3.0]
```
Didn't add a unit test because its a trivial change, but happy to add one at request.

**Review by felixfontein — COMMENTED** [review]
Looks good. This needs a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to), though.

**Comment by kaysond** [comment]
Added

File: changelogs/fragments/31-docker-secret.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "docker_secret - add a warning when the secret does not have an 'ansible_key' label but the 'force' parameter is not set"
```
Comment (felixfontein) [inline changelogs/fragments/31-docker-secret.yml:2]:
```suggestion
  - "docker_secret - add a warning when the secret does not have an ``ansible_key`` label but the ``force`` parameter is not set (https://github.com/ansible-collections/community.docker/issues/30, https://github.com/ansible-collections/community.docker/pull/31)."
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@kaysond thanks for improving this!

**Pull request merged**